openwhisk/actionProxy.go

```diff
@@ -72,6 +72,18 @@ func NewActionProxy(baseDir string, compiler string, outFile *os.File, errFile *
 
 //SetEnv sets the environment
 func (ap *ActionProxy) SetEnv(env map[string]interface{}) {
+	// Propagate proxy version
+	ap.env["__OW_PROXY_VERSION"] = Version
+	// propagate OW_EXECUTION_ENV as  __OW_EXECUTION_ENV
+	ee := os.Getenv("OW_EXECUTION_ENV")
```

Comment:
because the actions always uses the __OW variables while the proxy uses the OW variables. In short OW_VARIABLES are parameters for the proxy while the __OW are parameters for the action. It is all documented in the ENVVARS document I added